README.md

```diff
@@ -196,22 +197,21 @@ In `the cb_create` then just call the function for the management address and th
 class ServiceCallbacks(Service):
     @Service.create
     def cb_create(self, tctx, root, service, proplist):
-        self.log.info('Service create(service=', service._path, ')')
+        self.log.info('Service create(service=', service._path, ')') # pylint: disable=protected-access
 
         management_address = calculate_ip_address(service.management_prefix)
```

Comment:
I think it can be good to share to users which lines where modified. We can use diff inside markdown to showcase what was removed and added.

You can add which variables were simplified so is easier for users to spot.

Below is how markdown renders diffs and at the end you can see how it was created.

```diff
class ServiceCallbacks(Service):
    @Service.create
    def cb_create(self, tctx, root, service, proplist):
        self.log.info('Service create(service=', service._path, ')') # pylint: disable=protected-access

-       management_prefix = service.management_prefix
        self.log.debug(f'Value of management-prefix leaf is {management_prefix}')
-       net = ipaddress.IPv4Network(management_prefix)
-       management_address = list(net.hosts())[0]
+       management_address = calculate_ip_address(service.management_prefix)

-       bgp_prefix = service.bgp_prefix
        self.log.debug(f'Value of bgp-prefix leaf is {bgp_prefix}')
-       net = ipaddress.IPv4Network(bgp_prefix)
-       bgp_address = list(net.hosts())[0]
+       bgp_address = calculate_ip_address(service.bgp_prefix)

        tvars = ncs.template.Variables()
        tvars.add('MANAGEMENT_ADDRESS', management_address)
        tvars.add('BGP_ADDRESS', bgp_address)
        template = ncs.template.Template(service)
        template.apply('loopback-template', tvars)
```

below is the markdown block use
<pre>
```diff
class ServiceCallbacks(Service):
    @Service.create
    def cb_create(self, tctx, root, service, proplist):
        self.log.info('Service create(service=', service._path, ')') # pylint: disable=protected-access

-       management_prefix = service.management_prefix
        self.log.debug(f'Value of management-prefix leaf is {management_prefix}')
-       net = ipaddress.IPv4Network(management_prefix)
-       management_address = list(net.hosts())[0]
+       management_address = calculate_ip_address(service.management_prefix)

-       bgp_prefix = service.bgp_prefix
        self.log.debug(f'Value of bgp-prefix leaf is {bgp_prefix}')
-       net = ipaddress.IPv4Network(bgp_prefix)
-       bgp_address = list(net.hosts())[0]
+       bgp_address = calculate_ip_address(service.bgp_prefix)

        tvars = ncs.template.Variables()
        tvars.add('MANAGEMENT_ADDRESS', management_address)
        tvars.add('BGP_ADDRESS', bgp_address)
        template = ncs.template.Template(service)
        template.apply('loopback-template', tvars)
```
</pre>